core-ui/src/components/Predefined/Details/Ingress/RowComponent.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { FormItem, FormLabel } from 'fundamental-react';
+import './RowComponent.scss';
+
+export const RowComponent = ({ name, value }) => (
```

Comment:
I believe we can use `LayoutPanelRow` for that (well, it has no ':'). Besides, `item-wrapper` class is not defined anywhere - AFAIK it has been renamed to `break-word` (from root index.scss).